aws_tools/dynamodb_handler.py

```diff
@@ -90,27 +90,55 @@ def query_items(self, query=None, only_fields_with_values=True):
                     continue
 
                 if condition == 'eq':
-                    filter_expression &= Attr(field).eq(value)
+                    if filter_expression:
+                        filter_expression &= Attr(field).eq(value)
+                    else:
+                        filter_expression = Attr(field).eq(value)
```

Comment:
Instead of making all these changes, all you need to do is change `filter_expression = None` to `filter_expression = ''` above. An empty string is False, as is None.